development/lib/build-type.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * The distribution this build is intended for.
+ *
+ * This should be kept in-sync with the `BuildType` map in `shared/constants/app.js`.
```

Comment:
Is it possible to reuse that file instead of having to keep it in sync? I realize that that file is written in ESM syntax, so Node would not recognize it, but could we move this file into `shared/constants`?

Reply:
That would break our lint rules. We could add an exception for it though. Doesn't seem great to have mixed module formats scattered throughout though. At least now we have a simple division between the non-ESM scripts, and ESM everything else.

I was hoping we'd find a way to import ESM modules in Node.js. But I don't know what the status is of that these days.

Reply:
That's fair!